ztools/ztls/common.go

```diff
@@ -226,6 +226,10 @@ type ClientSessionCache interface {
 	Put(sessionKey string, cs *ClientSessionState)
 }
 
+type ClientExtension interface {
```

Comment:
This also should not be in this file- moving it to be next to the definition of `ClientFingerprintConfiguration`